### ISSUE TITLE
Urgent-discounts reflecting twice on invoices - SH010416

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#1346]
I'm going month by month in the DB to make sure information is correct and most of of the bookings on this tour are reflecting the discount twice so they're all showing negative balances which is wrong:

![image](https://cloud.githubusercontent.com/assets/8440903/17181041/9fd41346-53ec-11e6-9f06-642d1ca85137.png)

**Comment by bradatkins** [comment]
They seem to be showing 0 now. Either someone fixed the orders or I fixed the bug with some other fix?

**Comment by ali206** [comment]
?

![image](https://cloud.githubusercontent.com/assets/8440903/17776162/58c1e41c-6529-11e6-9eec-17c3842a6b00.png)

**Comment by bradatkins** [comment]
OK It's fixed; however, I want to make sure I didn't break other orders.

What happened was at some point in time were including the discount in the order line value.

Then we switched to showing it as a Payment.

For these orders, we were counting both. I'm not sure if this messed up other order's totals (I don't think so) and I'm not sure of the exact date we made this switch.

Let me know if you find others that are wrong.
